### PR TITLE
[zh]Sync pull-image-private-registry.md

### PR DESCRIPTION
**Pull request by mengjiao-liu**
Manually synchronize the `content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md ` file. 

After the `./scripts/lsync.sh` script is run, the synchronization content is inaccurate. It should be caused by a previous incomplete synchronization of the file.

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 1]
```diff
@@ -92,15 +98,77 @@ If you use a Docker credentials store, you won't see that `auth` entry but a `cr
...
+Be sure to:
+
+- set the name of the data item to `.dockerconfigjson`
+- base64 encode the Docker configuration file and then paste that string, unbroken
+  as the value for field `data[".dockerconfigjson"]`
+- set `type` to `kubernetes.io/dockerconfigjson`
+
+Example:
+-->
+如果你需要更多的设置（例如，在新 Secret 上设置名字空间或标签），
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md:134]:
```suggestion
如果你需要更多的设置（例如，为新 Secret 设置名字空间或标签），
```

File: content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 2]
```diff
@@ -92,15 +98,77 @@ If you use a Docker credentials store, you won't see that `auth` entry but a `cr
...
+则可以在存储 Secret 之前对它进行自定义。
+请务必：
+
+- 将 data 项的名称设置为`.dockerconfigjson`
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md:138]:
```suggestion
- 将 data 项中的名称设置为`.dockerconfigjson`
```

**Comment by mengjiao-liu** [comment]
All feedbacks addressed.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**